dbt/macros/tests/test_generate_schema_name.sql

```diff
@@ -5,13 +5,6 @@
     {% do test_generate_schema_name_raises_for_default_schema_name() %}
 {% endmacro %}
 
-{% macro mock_env_var(var_name) %}
-    {% if var_name == "USER" %} {{ return("test-user") }}
-    {% elif var_name == "HEAD_REF" %} {{ return("testuser/feature-branch-1") }}
-    {% else %} {{ return("") }}
-    {% endif %}
-{% endmacro %}
```

Comment:
This mock hasn't been deleted, I just moved it to the `utils` module below since I'm now reusing it outside the context of the `test_generate_schema_name` module.